devfile.yaml

```diff
@@ -21,14 +21,19 @@ components:
         port: 4000
         attributes:
           path: /che-7/overview/introduction-to-eclipse-che/
-  - id: errata-ai/vale-server/latest
-    type: chePlugin
+  - type: chePlugin
+    reference: https://che-plugin-registry-main.surge.sh/v3/plugins/errata-ai/vale-server/latest/meta.yaml
```

Comment:
It could be safer if we had deployment per release but at least here we ensure to use fresh stuff for dogfooding :-)